internal/components/components.go

```diff
@@ -85,6 +85,7 @@ import (
 	"github.com/signalfx/splunk-otel-collector/internal/extension/smartagentextension"
 	"github.com/signalfx/splunk-otel-collector/internal/receiver/databricksreceiver"
 	"github.com/signalfx/splunk-otel-collector/internal/receiver/smartagentreceiver"
+	timestampprocessor "github.com/signalfx/splunk-otel-collector/processor/timestampprocessor"
```

Comment:
nit, unnecessary named import

Reply:
OK, this hid a bigger problem. Good call. Fixing.